src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -29,12 +29,13 @@ class FilesystemLoader extends \Twig_Loader_Filesystem
     /**
      * Constructor.
      *
-     * @param FileLocatorInterface        $locator A FileLocatorInterface instance
-     * @param TemplateNameParserInterface $parser  A TemplateNameParserInterface instance
+     * @param FileLocatorInterface        $locator  A FileLocatorInterface instance
+     * @param TemplateNameParserInterface $parser   A TemplateNameParserInterface instance
+     * @param string                      $rootPath The project root path
      */
-    public function __construct(FileLocatorInterface $locator, TemplateNameParserInterface $parser)
+    public function __construct(FileLocatorInterface $locator, TemplateNameParserInterface $parser, $rootPath = '')
     {
-        parent::__construct(array());
+        parent::__construct(array(), realpath($rootPath));
```

Comment:
it should pass the rootPath without altering it. Thus, not all filesystems support realpath